README.md

```diff
@@ -1,34 +1,79 @@
-- Clone this repo, enter it
-- Run command `npm run setup`
-- _NB: To launch the backend server, you'll need an environment file with database credentials. You'll find a template one in `backend/.env.sample`_
+This WebApp is a music listening platform dedicated to education. It gives access to sung courses by subject (philosophy, chemistry, spanish...) and by level (college and high school) after connection.
+This website includes an administration interface for courses and users on React Admin.
 
-### Available Commands
+### Project status and progress
 
-- `setup` : Initialization of frontend and backend, as well as all toolings
-- `dev` : Starts both servers (frontend + backend) in one terminal
-- `dev-front` : Starts the React frontend server
-- `dev-back` : Starts the Express backend server
-- `lint` : Runs validation tools, and refuses unclean code (will be executed on every _commit_)
-- `fix` : Fixes linter errors (run it if `lint` growls on your code !)
+This project was developed by 6 developers in training at the Wild Code School of Lille (React Javascript) as an end of training project.
+The objective of this application for the client is to give to see and test an application close to what will be the final product, to make concrete the project and to give to see to the networks of sponsors and potential customers of WIM.
 
-## FAQ
+## Technology
 
-### Tools
+### Template
 
-- _Concurrently_ : Allows for several commands to run concurrently in the same CLI
-- _Husky_ : Allows to execute specific commands that trigger on _git_ events
-- _Vite_ : Alternative to _Create-React-App_, packaging less tools for a more fluid experience
-- _ESLint_ : "Quality of code" tool, ensures chosen rules will be enforced
-- _Prettier_ : "Quality of code" tool as well, focuses on the styleguide
-- _ Airbnb Standard_ : One of the most known "standards", even though it's not officially linked to ES/JS
-- _Nodemon_ : Allows to restart the server everytime a .js file is udated
+- Template : Application developed on React, Node Express from a [template](https://github.com/WildCodeSchool/js-template-fullstack) developped by the Wild Code School.
+
+**Tools used in this template:**
+
+- _Concurrently_ : Allows to execute several commands in the same terminal
+- _Husky_ : Allows to execute actions when triggered by _git_ commands
+- _Vite_ : Alternative to _Create-React-App_, with fewer packages for a smoother experience
+- _ESLint_ : "Code quality" tool, ensures that pre-configured rules are respected
+- _Prettier_ : "code quality" tool, also focuses on code style
+- _Standard Airbnb_ : One of the most famous "standards", even if it is not officially linked to ES/JS
+- _Nodemon_ : Tool to restart a server each time one of the files is modified
+
+### Other libraries
+
+- MySQL: A database has been developed on mysql thanks to WorkBench
+- ReactAdmin` : Course and user management has been developed on React Admin
```

Comment:
Broken line

Reply:
I don't understand "broken line" : it means there is a blank line I must delete? (I put one after every title) or something else? (in which case I don't understand what)